st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -86,32 +85,35 @@ class ActionExecutionsControllerMixin(BaseRestControllerMixin):
         'trigger_instance'
     ]
 
-    def _get_from_model_kwargs_for_request(self, request):
+    def _get_from_model_kwargs_for_request(self, **kwargs):
         """
         Set mask_secrets=False if the user is an admin and provided ?show_secrets=True query param.
         """
-        return {'mask_secrets': self._get_mask_secrets(request)}
+        return {'mask_secrets': not kwargs.get('show_secrets', False)}
 
-    def _handle_schedule_execution(self, liveaction_api):
+    def _handle_schedule_execution(self, liveaction_api, requester_user=None, **kwargs):
         """
         :param liveaction: LiveActionAPI object.
         :type liveaction: :class:`LiveActionAPI`
         """
 
+        if not requester_user:
```

Comment:
As I said, I see it more explicit this way. Having 2-line utility method instead of a simple if makes reading code harder and requires you fo constantly jump between the files.